content/en/shift_left.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Shift Left
+status: Feedback Appreciated
+category: Concept
+---
+
+## What it is
+Shift Left is the practice of implementing tests, security, or other development practices early in the software development cycle rather than towards the end. Although originally used to refer to the process of testing early, Shift Left can now also be applied to other aspects of software development and [DevOps](https://glossary.cncf.io/devops/), such as security and deployment. 
```

Comment:
Love it! Very easy to understand. We should explain why it's called "shift left" though as not everyone is familiar with the concept of code moving from left to right

Reply:
Added a couple sentences based on the note....let me know if this helps to cover this topic correctly!